apps/governance/src/routes/proposals/components/proposals-list/proposals-list.tsx

```diff
@@ -54,13 +59,30 @@ export const orderByUpgradeBlockHeight = (
     ['desc', 'desc']
   );
 
+enum ClosedProposalsViewOptions {
+  NetworkGovernance = 'networkGovernance',
+  NetworkUpgrades = 'networkUpgrades',
+}
+
 export const ProposalsList = ({
   proposals,
   protocolUpgradeProposals,
   lastBlockHeight,
 }: ProposalsListProps) => {
   const { t } = useTranslation();
   const [filterString, setFilterString] = useState('');
+  const [closedProposalsView, setClosedProposalsView] =
+    useState<ClosedProposalsViewOptions>(
+      ClosedProposalsViewOptions.NetworkGovernance
+    );
+
+  useEffect(() => {
```

Comment:
no reason for this affect. Call `setClosedProposalsView` directly where needed

```
<ProposalsListFilter
      filterString={filterString}
      setFilterString={(value) => {
        setFilterString(value)
        if (value.length > 0) {
          setClosedProposalsView(ClosedProposalsViewOptions.NetworkGovernance)
        }
      }}
    />
```

See: https://react.dev/learn/you-might-not-need-an-effect

Reply:
Ah damnit, fell into that trap! Thanks